routers/web/user/home.go

```diff
@@ -444,21 +444,6 @@ func buildIssueOverview(ctx *context.Context, unitType unit.Type) {
 		repoOpts.TeamID = team.ID
 	}
 
-	switch filterMode {
```

Comment:
I don't know whether we need to remove `repoOpts` above in this PR. it is not used.